config/routes.rb

```diff
@@ -28,6 +28,10 @@
         end
       end
       get 'create_key' => 'products#create_key'
+      get 'new_from_git_remote' => 'products#new_from_git_remote'
+      get 'new_from_rails_template' => 'products#new_from_rails_template'
+      post 'create_from_git_remote' => 'products#create_from_git_remote'
+      post 'create_from_rails_template' => 'products#create_from_rails_template'
```

Comment:
Before:
```         
                create_key GET    /admin/:type/products/create_key(.:format)                               products#create_key {:type=>/dev|usr/}
       new_from_git_remote GET    /admin/:type/products/new_from_git_remote(.:format)                      products#new_from_git {:type=>/dev|usr/}
   new_from_rails_template GET    /admin/:type/products/new_from_rails_template(.:format)                  products#new_from_rails_template {:type=>/dev|usr/}
    create_from_git_remote POST   /admin/:type/products/create_from_git_remote(.:format)                   products#create_from_git_remote {:type=>/dev|usr/}
create_from_rails_template POST   /admin/:type/products/create_from_rails_template(.:format)               products#create_from_rails_template {:type=>/dev|usr/}
```

Note that doing this as a `collection do` block appends `_products` to each of the routes.